worker/worker.go

```diff
@@ -0,0 +1,60 @@
+package worker
+
+import (
+	"encoding/json"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"io"
+	"log"
+)
+
+type Worker struct {
```

Comment:
we need internal doc explaining this struct. what is token? what is ram? why do we use strings for vcpu and ram (it seems we are going to keep converting them during runtime if we use them to anything more than logging). should Id be a string?

Reply:
I mean, this struct definition is really important to our codebase

Reply:
I think the only think that makes sense to be a string in this case is Id, i'll change ram and vcpu to  float..